web/src/styles/global.tsx

```diff
@@ -1,12 +1,29 @@
 import { createUseStyles } from 'react-jss'
 import tokens from '../tokens'
 
+const focusStyle = ({ isInvalid }): string => {
```

Comment:
Should we use `outline` here instead of `boxShadow`?

Reply:
outlines look strange if border-radius is set. using `boxShadow` as focus style is very common. check out the input focus from github :)

Reply:
I see, well then I'll switch to `boxShadow` too